templates/emails/user.monthlyreport.hbs

```diff
@@ -226,7 +226,7 @@ Subject: {{capitalize month}} Report
           {{#if consolidatedPdfs}}<br />You can find PDFs of all your contributions, organized by fiscal host, attached to this email.{{/if}}</li>
           {{#each collectives}}
             {{#if nextGoal}}
-              <li><a href="{{../config.host.website}}/{{slug}}">{{name}}</a> needs {{currency nextGoal.missing.amount currency=currency}} more {{#if nextGoal.missing.interval}}per {{nextGoal.missing.interval}}{{/if}} to reach their next goal of {{nextGoal.title}} ({{nextGoal.percentage}} there). 
+              <li><a href="{{../config.host.website}}/{{slug}}">{{name}}</a> needs {{currency nextGoal.missing.amount currency=currency}} more {{#if nextGoal.missing.interval}} {{#ifCond nextGoal.missing.interval '!==' 'flexible'}}per {{nextGoal.missing.interval}} {{/ifCond}} {{/if}} to reach their next goal of {{nextGoal.title}} ({{nextGoal.percentage}} there). 
```

Comment:
My understanding from https://github.com/opencollective/opencollective-api/blob/87629f67ba47df6fad7dcb478f01a6fc9d803c51/server/models/Collective.js#L722 is that it can only be `month` or `year`, not `flexible`